packages/frontend/core/src/blocksuite/view-extensions/code-block-preview/mermaid-preview.ts

```
@@ -0,0 +1,515 @@
+      );
+    }
+
+    return result;
+  }
+
+  override updated(changedProperties: PropertyValues): void {
+    const result = super.updated(changedProperties);
+    if (changedProperties.has('mermaidCode')) {
+      this._scheduleRender();
+    }
+    return result;
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)